plugins/module_utils/foreman_helper.py

```diff
@@ -14,6 +14,7 @@
 from functools import wraps
 
 from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils._text import to_bytes
```

Comment:
Is this meant seriously? I need a private interface to do the simple things in 'the right way'?